sumo-api/sumo/metrics/utils.py

```diff
@@ -337,16 +378,14 @@ def nearest_neighbors(points1, points2):
     return ind1to2, ind2to1, dist1to2, dist2to1
 
 
-def points_rmsssd(evaluator, submission, ground_truth, overlap_thresh, voxels=False):
+def points_rmsssd(evaluator, submission, ground_truth, voxels=False):
```

Comment:
Why are you removing the overlap_thresh check here?

Reply:
I explain this in the relevant commit description bb4dc1a  "Overlap threshold should only be used for similarity computation. After a match is found, all points should be considered to compute the rms error.". See relevant equation.

Reply:
I see.  The commit descriptions do not show up in the review summary, so I did not see them.  Here is the problem with removing the threshold...  The goal of the RMS computation is typically to determine how well the shape matches in the area where the shapes correspond.  The goal of the IoU computation (for meshes and voxels) is to determine how much of the surfaces correspond.  These are two complimentary metrics that give different types of information about the modeling.  By removing the threshold, the areas where the shapes do not correspond will be counted as errors.  These errors will typically be much larger than the errors in areas where the surfaces correspond, so the meaningful errors will be dwarfed by the non-meaningful errors.  In such a case, the RMS metric will not be complimentary to the IoU computation and will not be useful.  I suggest that we leave the threshold in.

Reply:
The IOU measures the percentage similarity to establish matches - a match is established when the percent of similar surfaces are more than the dissimilar surfaces as determined by the similarity threshold. 
Because the rms error is only computed on matches beyond the IOU threshold - meaning the similar surfaces are more than the dissimilar surfaces, the dissimilar surfaces shouldn't dominate on average.
Once a match is established by IOU, the RMS error measures the error of that match. For instance if we have 2 submitted shapes with same IOU greater than the threshold, the RMS error helps break the tie to determine which submission is better. ie a good enough chair with 95% IOU and a gaping defect, should get a lower score than the same chair with IOU 95% where the defect is smaller. The IOU does not account for that and thresholding rms would give those 2 chairs the same score.